src/dbl.ml

```diff
@@ -28,7 +28,15 @@ let cmd_args_options = Arg.align
 
     "-verbose-internal-errors",
     Arg.Set InterpLib.InternalError.verbose,
-    " Make internal errors more verbose (for debugging only)"
+    " Make internal errors more verbose (for debugging only)";
+
+    "-I",
+    Arg.String (fun p -> Config.lib_search_dirs := !Config.lib_search_dirs @ [p]),
```

Comment:
Expressions like `xs @ [x]` performed in a loop tend to look a bit fishy (though, to be fair, these lists are unlikely to be long). Besides, I'm not sure that we actually want to add directories from command-line options at the very end; perhaps they should take precedence over some other potential sources (_e.g.,_ environment variables), even if presently it makes no difference. I think you can solve both of these minor issues by accumulating these directories in reverse order into a temporary list, and `List.rev_append`ing them with the list in `Config` after parsing all arguments.

Also, this line is too long.